kano_world/hooks/launch.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+# launch.py
+#
+# Copyright (C) 2014, 2015 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
+#
+
+# args is an array of arguments that were passed through the URL
+# e.g., kano:launch:make-art, it will be ["make-art"]
+
+import os
+import sys
+
+if __name__ == '__main__' and __package__ is None:
+    dir_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
+    if dir_path != '/usr':
+        sys.path.insert(1, dir_path)
+
+from kano.utils import run_cmd
+
+
+def run(args):
+    try:
+        appName = args[0]
+
+        if (appName == 'make-art'):
+            appName = 'kano-draw'
+
+        if (appName == 'make-snake'):
+            appName = 'kano-launcher \"rxvt -title \\\"Make Snake\\\" -e python /usr/share/make-snake'
+
+        if (appName == 'terminal-quest'):
+            appName = 'linux-story-gui'
```

Comment:
@Ealdwulf That is true. Something like what is done in [`launch_project `](https://github.com/KanoComputing/kano-profile/blob/eda34dbbe93a0dc31278449e6129b15b9f3522dc/kano_profile/apps.py#L124-L136) but whatever we do, we should ensure that arbitrary commands can't be executed by using this hook.